### PR TITLE
Staging: Getting started section restructure + Fares data samples

### PR DESCRIPTION
**Pull request by Sergiodero**
These changes consist of:

- Restructuring the Getting started section into multiple subsections
- Fixing Fares features data samples

This does not yet include the copy udpate for the new subsections, this is to be addressed in a new PR.

@eliasmbd could you review the new structure for the Getting started section?
@tzujenchanmbd could you take a look at the Fares data samples?

Thanks!

### CODE REVIEW

**Review by eliasmbd — COMMENTED** [review]
LGTM, The structure is good. Once we re-do some of the content we might have to make some changes but for now it works

File: docs/getting_started/features/fares.md [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+        | clipper       | Clipper          | 2               |
+        | munimobile    | SFMTA MuniMobile | 4               |
+        | cash          | Cash             | 0               |  
+
+    !!! note ""
+        <p style="font-size:16px">
+        <a href="/schedule/reference/#fare_productstxt"><b>fare_products.txt</b></a> <br>
+        </p>
+
+        | fare_product_id  |
```
Comment (tzujenchanmbd) [inline docs/getting_started/features/fares.md]: Probably need to include media here

File: docs/getting_started/features/fares.md [hunk 2]
```diff
@@ -0,0 +1,304 @@
+# Fares
+GTFS allows to preciely model a wide variety of fare structures used by different transit agencies around the world, such as fares based by zone, by distance traveled, or by time-of-day. GTFS Fares informs riders of the price applicable to their trip and the media they can use to pay.
+
+## Fare Products
+
+Fare Products lists the types of tickets or fares (i.e. single-trip fare, monthly pass, etc.) offered by a transit agency to access a service. Fare Products serve as a foundation for modeling an agency's fare structure, and they are linked to the transit service through mechanisms outlined in `fare_leg_rules.txt`. The association of Fare Products to various travel conditions, such as routes, areas, and times, determines the fare costs for individual travel segments and transfers.
```
Comment (tzujenchanmbd) [inline docs/getting_started/features/fares.md]: Perhaps mention virtual entities like transfer costs here?

Comment (Sergiodero) [reply docs/getting_started/features/fares.md]: Added "transfer fees" as an additional example

File: docs/getting_started/features/fares.md (hunk 1 above)
Comment (Sergiodero) [reply docs/getting_started/features/fares.md]: Included all required fields for fare_products.txt in addition to fare_media

**Comment by Sergiodero** [comment]
Thank you both! changes were made in a last commit to address @tzujenchanmbd comments, I'll merge the content now

**Pull request merged**